### PR TITLE
indicator_1-2-1

### PR DESCRIPTION
**Pull request by bishopkws**

### CODE REVIEW

File: meta/1-2-1.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+international_and_national_references: ''
+time_period: 2000 - 2022
+scheduled_update_by_SDG_team: ''
+source_organisation_1: ''
+source_agency_staff_name_1: ''
+source_agency_survey_dataset_1: ''
+source_notes_1: ''
+source_title_1: ''
+source_url_1: >
+  https://www2.census.gov/programs-surveys/cps/tables/time-series/historical-poverty-people/hstpov2.xlsx
```
Comment (btylerburton) [inline meta/1-2-1.yml:36]: is the spacing here correct?

**Comment by bishopkws** [comment]
I don’t think so. In the yml file, that web link is under DATA COMP and
appears in the middle of the file. I’m u sure how to make sure it uploads
properly.

**Comment by btylerburton** [comment]
I don't see any other examples of `DATA_COMP` in the repo, and all `source_url_1` are single http strings...

See here: https://github.com/GSA/sdg-data-usa-staging/blob/main/meta/13-3-2.md?plain=1#L23

@brockfanning are you aware of what format we want here?

Can we add `source_url_2` and a new `source_url_3` and print each URL individually?

**Comment by brockfanning** [comment]
@btylerburton You're right that that field is not going to render as expected there. But feel free to merge this. We can work with the data provider to get it closer to what they want.

@bishopkws I'll follow up about possible approaches afterwards.

**Review by btylerburton — APPROVED** [review]

**Pull request merged**